### PR TITLE
finished

### PR DESCRIPTION
**Pull request by barantarasNEW**
https://barantarasNEW.github.io/layout_catalog_grid/

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,27 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@mixin responsive($minWidth, $howRepeat) {
+  @media (min-width: #{$minWidth}) {
+    grid-template-columns: repeat($howRepeat, minmax(200px, 1fr));
```
Comment (ivanchuk-yurii) [inline src/styles/_catalog.scss:3]:
```suggestion
    grid-template-columns: repeat($howRepeat, 200px);
```
bc now it looks wrong, move 200px into variable
image

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,22 +1,27 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@mixin responsive($minWidth, $howRepeat) {
+  @media (min-width: #{$minWidth}) {
+    grid-template-columns: repeat($howRepeat, 200px);
```
Comment (ivanchuk-yurii) [inline src/styles/_catalog.scss:3]: move `200px` into variable

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
should be centered, check demo before requesting new review 
image

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,22 +1,30 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$widthCard: 200px;
 
+@mixin responsive($minWidth, $howRepeat) {
```
Comment (Viktor-Kost) [inline src/styles/_catalog.scss:3]: It's not obvious to use mixins like this
I call @include responsive, ok i pass $minWidth, it's ok. 
But $howRepeat does not applies to responsive. It's about grid. 
So what if i would like to have another 5-10 different styles that have to be changed inside responsive mixin?
Please, rewrite without using mixin like this.

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hi! Check my comments

**Review by OlEzhA0 — APPROVED** [review]
Ok, I approve without the change that I left last time (try to use `@each`)
But anyway it's good practice for you to try

**Pull request closed without merging**